StellarDotnetSdk/Soroban/SCVal.cs

```diff
@@ -1342,13 +1342,13 @@ public ContractExecutableWasm(string hash)
     }
 
     /// <summary>
-    ///     Base-64 encoded string of SHA-256 hash of a previously uploaded Wasm.
+    ///     Hex encoded string of SHA-256 hash of a previously uploaded Wasm bytes.
     /// </summary>
     public string WasmHash { get; }
```

Comment:
Is the wasmHash the hex-encoded Wasm ID or hex-encoded wasmHash?  Or is the wasmId a SHA-256 hash of Wasm bytes? Getting a bit confused now if I compare this description to the previous one.

Reply:
Maybe you can help me with the wording for this.
So, Wasm ID or Wasm hash, or just hash, are used interchangeably, and they refer to the hash of an uploaded Wasm bytes. This (SHA256) hash is represented as a hex-encoded string.
The term "Wasm bytes" is used in the docs here:
https://developers.stellar.org/docs/build/smart-contracts/getting-started/deploy-increment-contract#two-step-deployment
I hope it's clearer now.

Reply:
Like you said in the official docs they call it the `washHash` and they say `hash of the Wasm bytes` yes I agree we should remove `wasmId` completely.

Extra proof is that we see no results when looking for [wasmId](https://www.google.com/search?q=site%3Adevelopers.stellar.org+%22wasmId%22) but we do find results for [wasmHash](https://www.google.com/search?q=site%3Adevelopers.stellar.org+%22wasmHash%22).

But I would change the docs of the `wasmHash` simply to `A hex-encoded value of the Wasm bytes`.

I think you got the term from the java SDK it is still in there. But I think what you changed is better than that. Please update all the `wasmHash` docs with the simple version.

Reply:
Done.